packages/primereact/src/SelectWidget/SelectWidget.tsx

```diff
@@ -0,0 +1,141 @@
+import { FocusEvent } from 'react';
+import { Dropdown } from 'primereact/dropdown';
+import {
+  ariaDescribedByIds,
+  enumOptionsIndexForValue,
+  enumOptionsValueForIndex,
+  FormContextType,
+  RJSFSchema,
+  StrictRJSFSchema,
+  WidgetProps,
+} from '@rjsf/utils';
+import { MultiSelect } from 'primereact/multiselect';
+
+/** The `SelectWidget` is a widget for rendering dropdowns.
+ *  It is typically used with string properties constrained with enum options.
+ *
+ * @param props - The `WidgetProps` for this component
+ */
+export default function SelectWidget<T = any, S extends StrictRJSFSchema = RJSFSchema, F extends FormContextType = any>(
+  props: WidgetProps<T, S, F>,
+) {
+  const multiple = typeof props.multiple === 'undefined' ? false : props.multiple;
```

Comment:
This is easier to read and shorter
```suggestion
  const { multiple = false } = props;
```